test/circle/lint.sh

```diff
@@ -30,6 +44,28 @@ for directory in ${CHANGED_FOLDERS}; do
   if [ "$directory" == "incubator/common" ]; then
     continue
   elif [ -d $directory ]; then
-    helm lint ${directory}
+    run helm lint ${directory}
+
+    # If a Chart.yaml file is present lint it. Otherwise report an error
+    # because one should exist
+    if [ -e ${directory}/Chart.yaml ]
```

Comment:
I'd prefer if you stick to the existing style and put `then` on the same line.

`if [ -e ${directory}/Chart.yaml ]; then`